media-sound/synthv1/synthv1-9999.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit fdo-mime
+
+DESCRIPTION="Old-school all-digital 4-oscillator subtractive polyphonic synthesizer"
+HOMEPAGE="http://synthv1.sourceforge.net"
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3 autotools
+	EGIT_REPO_URI="https://github.com/rncbc/${PN}.git"
+	KEYWORDS=""
+else
+	SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+LICENSE="GPL-2+"
+SLOT="0"
+
+IUSE="debug qt5 standalone alsa lv2 osc"
+REQUIRED_USE="
+	|| ( standalone lv2 )
+	alsa? ( standalone )"
+
+RDEPEND="
+	!qt5? (
+		dev-qt/qtcore:4
+		dev-qt/qtgui:4
+	)
+	qt5? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtwidgets:5
+		dev-qt/qtxml:5
+	)
+	media-libs/libsndfile
+	standalone? ( virtual/jack )
+	alsa? ( media-libs/alsa-lib )
+	lv2? ( media-libs/lv2 )
+	osc? ( media-libs/liblo )
+"
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	# Fix to ensure this desktop file is used to get the application's icon
```

Comment:
This works, for now it can stay here until it gets fixed upstream.
Then it would either be needed to move it into an if for !9999 or the ebuilds should be split